packages/web/src/components/data-entry/AccessAndSaleTriggerLegacy.tsx

```diff
@@ -34,9 +34,15 @@ import { AccessAndSaleMenuFields } from 'pages/upload-page/fields/AccessAndSaleM
 import { getCombinedDefaultGatedConditionValues } from 'pages/upload-page/fields/helpers'
 import {
   AccessAndSaleFormValues,
+  DOWNLOAD_CONDITIONS,
+  // DOWNLOAD_REQUIRES_FOLLOW,
```

Comment:
oh. forgot to verify this current behavior (when loss feature flag is false). getting to it asap